app/Http/Controllers/PrintController.php

```diff
@@ -52,6 +52,26 @@ public function print(Request $request) {
             return back()->withErrors(['print' => __('print.error_printing')]);
         }
     }
+    public function sendPages(Request $request) {
+        $validator = Validator::make($request->all(), [
+            'balance' => 'required|integer',
```

Comment:
I don't think a negative balance transfer should be allowed.
Validate the proper value range.